clients/venice-thin-client/src/main/java/com/linkedin/venice/client/store/AbstractAvroComputeRequestBuilder.java

```diff
@@ -204,12 +203,6 @@ protected Map<String, Object> getCommonComputeSpec() {
    * @return a set of existing operations result field name
    */
   protected Set<String> commonValidityCheck() {
-    // Projection
-    projectFields.forEach(projectField -> {
```

Comment:
I thought about this logic a little more, and maybe we should rely on the newly introduced config to decide whether client should throw exception or not for non-existing project fields.
For PYMK, it is unexpected to project non-existing fields since they are using superset schema.

Reply:
Ok, will change this.